README.md

```diff
@@ -203,6 +203,7 @@ Mysql2::Client.new(
   :reconnect = true/false,
   :local_infile = true/false,
   :secure_auth = true/false,
+  :ssl_mode = "DISABLED" | "PREFERRED" | "REQUIRED" | "VERIFY_CA" | "VERIFY_IDENTITY",
```

Comment:
The Flags above are constant integers and can be OR-ed together. Based on http://dev.mysql.com/doc/refman/5.7/en/secure-connection-options.html#option_general_ssl it looks like these are mutually-exclusive values -- e.g. VERIFY_CA implied REQUIRED and does not need both options to be specified?